src/components/DownloadComponents/DLCard/DLCard.tsx

```diff
@@ -0,0 +1,136 @@
+import { FC } from "react";
+import styled from "styled-components";
+import { useHover } from "react-aria";
+import { capitalize } from "lodash";
+
+import type { DownloadResourceType } from "../downloads.types";
+
+import { RESOURCE_TYPE_ICON_MAP } from "@/components/DownloadComponents/DownloadCard/DownloadCard";
+import BoxBorders from "@/components/SpriteSheet/BrushSvgs/BoxBorders";
+import P from "@/components/Typography/P";
+import Flex from "@/components/Flex";
+import Checkbox from "@/components/Checkbox";
+import Icon from "@/components/Icon";
+import { CheckboxProps } from "@/components/Checkbox/Checkbox";
+
+export type DownloadCardProps = CheckboxProps & {
+  label: string;
+  resourceType: DownloadResourceType;
+  extension: string;
+  hasError?: boolean;
+};
+
+type DownloadCardLabelProps = DownloadCardProps & {
+  isHovered: boolean;
+};
+
+const BoxWithFocusState = styled.div`
+  position: relative;
+  border: solid 4px transparent;
+  /* cursor: pointer; */
+  display: flex;
+  flex-direction: "row";
+`;
+
+const DownloadCardLabel: FC<DownloadCardLabelProps> = ({
+  isHovered,
+  resourceType,
+  label,
+  extension,
+}) => {
+  return (
+    <BoxWithFocusState>
+      <Flex
+        $alignItems={"center"}
+        $justifyContent={"center"}
+        $pa={8}
+        $background={"teachersYellow"}
+        $width={66}
+      >
+        <Icon
+          name={RESOURCE_TYPE_ICON_MAP[resourceType]}
+          $display={"flex"}
+          $pa={8}
+          height={50}
+          $width={50}
+          $objectPosition={"center"}
+        />
+      </Flex>
+
+      <BoxBorders gapPosition="rightTop" />
+      <Flex
+        $flexDirection="column"
+        $alignItems="center"
+        $width={["100%"]}
+        $maxWidth={320}
+      >
+        <Flex
+          $flexDirection="column"
+          $justifyContent={"center"}
+          $ph={16}
+          $pv={16}
+          $width={"100%"}
+        >
+          <P
+            $font="heading-7"
+            $mb={4}
+            $textDecoration={isHovered ? "underline" : "none"}
+          >
+            {label}
+          </P>
+          <P $color={"oakGrey4"}>{capitalize(extension)}</P>
+        </Flex>
+      </Flex>
+    </BoxWithFocusState>
+  );
+};
+
+const DownloadCard: FC<DownloadCardProps> = (props) => {
+  const {
+    checked = false,
+    onChange,
+    id,
+    name,
+    label,
+    onBlur,
+    hasError = false,
+  } = props;
+
+  const { hoverProps, isHovered } = useHover({});
+
+  return (
+    <>
+      {hasError && (
+        <Flex>
+          <Icon name="content-guidance" $color={"teachersRed"} $width={24} />
+          <P $color={"teachersRed"} $font={"body-2"}>
+            error message
```

Comment:
should this be the actual error? it also seems to be sitting within the card at the mo 
<img width="352" alt="Screenshot 2023-10-25 at 13 35 45" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/f49d3a41-706a-4350-b845-eb9488357cff">